src/source/PeerConnection/PeerConnection.h

```diff
@@ -134,6 +144,7 @@ VOID onSctpSessionDataChannelOpen(UINT64, UINT32, PBYTE, UINT32);
 
 STATUS sendPacketToRtpReceiver(PKvsPeerConnection, PBYTE, UINT32);
 STATUS changePeerConnectionState(PKvsPeerConnection, RTC_PEER_CONNECTION_STATE);
+STATUS twccManagerOnPacketSent(PKvsPeerConnection pc, PRtpPacket pRtpPacket);
```

Comment:
Nit: 
`STATUS twccManagerOnPacketSent(PKvsPeerConnection pc, PRtpPacket pRtpPacket);` to `STATUS twccManagerOnPacketSent(PKvsPeerConnection, PRtpPacket);`